test/integration/source_openssl/openssl_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe directory '/usr/local/ssl/' do
+  it { should exist }
+end
+
+describe command('haproxy -vv | grep OpenSSL') do
+  its('stdout') { should match /Built with OpenSSL version/ }
```

Comment:
Is there a way to to ensure it's actually built against the correct version in this test?